lib/matplotlib/figure.py

```diff
@@ -1322,6 +1322,10 @@ def subplots_adjust(self, left=None, bottom=None, right=None, top=None,
         Unset parameters are left unmodified; initial values are given by
         :rc:`figure.subplot.[name]`.
 
+        The following plot illustrates the effect of the parameters:
+
```

Comment:
```suggestion
```

Suggest you just remove this for now.  It should be obvious what the plot is for, and is less distracting for folks who can't see the plot.

Reply:
ok